### ISSUE TITLE
Add license per project

### ISSUE THREAD

**Issue by jiho** [ecotaxa/ecotaxa_dev#511]
Add a field in project settings with the following choices

```
CC-0
CC-BY 4.0
CC-BY-NC 4.0
Copyright
```

In the help text, specify

[**CC-0**](https://creativecommons.org/publicdomain/zero/1.0/): users are free to redistribute, modify, and build upon the data downloaded, with no conditions. The data falls into the worldwide public domain. This is the license preferred by [OBIS](https://obis.org/manual/policy/) and [GBIF](https://www.gbif.org/terms).

[**CC-BY**](https://creativecommons.org/licenses/by/4.0/): users are free to redistribute, modify, and build upon the data downloaded, as long as they cite the dataset and its authors.

[**CC-BY-NC**](https://creativecommons.org/licenses/by-nc/4.0/): users are free to redistribute, modify, and build upon the data downloaded, as long as they cite the dataset and its authors, and do not use it for commercial purpose ("primarily intended for or directed toward commercial advantage or monetary compensation").

**Copyright**: only contributors to this project have rights on this data. This prevents its distribution in any kind of database.

NB: the full choices of CC licences is

```
CC-0
CC-BY 4.0
CC-BY-SA 4.0
CC-BY-NC 4.0
CC-BY-NC-SA 4.0
CC-BY-ND 4.0
CC-BY-NC-ND 4.0
```

But:
- SA (Share Alike) is non enforcable for our purpose and therefore useless.
- ND means that people cannot do anything with the data and therefore useless.
- OBIS and GBIF only supports the choices above (minus copyright) https://obis.org/manual/policy/

**Comment by picheral** [comment]
Replace Copyright: only contributors to this project have rights on this data. This prevents its distribution in any kind of database.
by Copyright: only **annotators** to this project have rights on this data. This prevents its distribution in any kind of database.

**Comment by jiho** [comment]
Need to

- [ ] reduce the current choice of licenses to the four above
- [ ] move the license in its own "line" in the project settings (not next to "Visible" but above it)
- [ ] add the help text next to it, like the rest of the help texts on the page
- [ ] default to no choice except for the ones that are already CC-something

**Comment by grololo06** [comment]
Fixed in 2.5.2.

**Comment by jiho** [comment]
Need some help text on the page. No one reads the CC licenses. Here is a possibly even more explicit version


[**CC-0**](https://creativecommons.org/publicdomain/zero/1.0/): all registered EcoTaxa users are free to download, redistribute, modify, and build upon the data, with no conditions. Other databases can index the data. The data falls into the worldwide public domain. This is the license preferred by [OBIS](https://obis.org/manual/policy/) and [GBIF](https://www.gbif.org/terms).

[**CC-BY**](https://creativecommons.org/licenses/by/4.0/): all registered EcoTaxa users are free to download, redistribute, modify, and build upon the data, as long as they cite the dataset and its authors. Other databases can index the data.

[**CC-BY-NC**](https://creativecommons.org/licenses/by-nc/4.0/): all registered EcoTaxa users are free to download, redistribute, modify, and build upon the data, as long as they cite the dataset and its authors, and do not use it for commercial purpose ("primarily intended for or directed toward commercial advantage or monetary compensation"). Other databases can index the data.

**Copyright**: only contributors to this project have rights on this data. This prevents its distribution in any kind of database.

**Comment by grololo06** [comment]
Fixed in 2.5.2.

**Comment by grololo06** [comment]
Licenses text hotfixed onto production site.
